docs/algorithms/sig/mayo.md

```diff
@@ -0,0 +1,62 @@
+# MAYO
+
+- **Algorithm type**: Digital signature scheme.
+- **Main cryptographic assumption**: multivariable quadratic equations, oil and vinegar.
+- **Principal submitters**: Ward Beullens, Fabio Campos, Sofía Celi, Basil Hess, Matthias J. Kannwischer.
+- **Authors' website**: https://pqmayo.org
+- **Specification version**: https://eprint.iacr.org/2023/1683.
```

Comment:
Conceptually, reference to a paper is unique & unmutable and thus useful as version ID. But when looking at the URL, it points to several revisions of the same paper. At first glance there seems to be no guarantee that the paper & the spec within cannot change anymore -- which would bring (this) implementation copy and spec out of line if/as when put into a library release. So the question: Do the Mayo authors guarantee that there will be no more paper updates now? Without that, I'd strongly suggest to change this to a "traditional" version ID.

Reply:
Perhaps this would be resolved by linking instead to the specific revision on the IACR eprint: https://eprint.iacr.org/archive/2023/1683/20240115:123013. To the best of my knowledge, the IACR maintains the version history as immutable.

Reply:
Good point. The reference was to the preprint on eprint. I updated the reference to the published version in TCHES with its unique doi: https://doi.org/10.46586/tches.v2024.i2.252-275